<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="Panyam-Win10" name="Com.Test.BlueBadge.Test.LoginPageTest" tests="6" failures="1" timestamp="13 Apr 2020 18:24:57 GMT" time="24.971" errors="0">
  <testcase name="verifyPageHeaderTest" time="0.620" classname="Com.Test.BlueBadge.Test.LoginPageTest"/>
  <testcase name="verifyCancelButtonPresenceTest" time="0.660" classname="Com.Test.BlueBadge.Test.LoginPageTest"/>
  <testcase name="verifyregisterButtonPresenceTest" time="0.612" classname="Com.Test.BlueBadge.Test.LoginPageTest"/>
  <testcase name="verifyForgotPasswordLink" time="23.079" classname="Com.Test.BlueBadge.Test.LoginPageTest">
    <failure type="java.lang.AssertionError" message="expected [Reset My Password] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [Reset My Password] but found [null]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Com.Test.BlueBadge.Test.LoginPageTest.verifyForgotPasswordLink(LoginPageTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyForgotPasswordLink -->
  <testcase name="doLoginTest" classname="Com.Test.BlueBadge.Test.LoginPageTest">
    <skipped/>
  </testcase> <!-- doLoginTest -->
  <testcase name="doLogin_InvalidDetails" classname="Com.Test.BlueBadge.Test.LoginPageTest">
    <skipped/>
  </testcase> <!-- doLogin_InvalidDetails -->
</testsuite> <!-- Com.Test.BlueBadge.Test.LoginPageTest -->
